extract/dcinside/main.py

```
@@ -0,0 +1,600 @@
+        # post_date = str(md_to_ymd(post_info['date']))
+        file_path = f"extracted/{self.car_id}/{self.folder_date}/{self.batch}/raw/dcinside/{post_info['id']}.json"
+        directory = os.path.dirname(file_path)
+
+        
+        # if not os.path.exists(directory):  # 디렉토리가 존재하지 않으면
+        #     os.makedirs(directory)  # 디렉토리 생성
+        # 1. 폴더 존재 확인
+        try:
+            self.s3.head_object(Bucket=self.BUCKET_NAME, Key=directory)
+            print("✅ S3 folder route exists")
+        except:  # 폴더가 없는 경우
```

Ruff (0.8.2)

Do not use bare `except`

(E722)